app/i18n/messages/en.json

```diff
@@ -11,7 +11,7 @@
   "AboutPageContent.espooPartnersHeader": "Espoo is piloting Varaamo as a part of the 6Aika project.",
   "AboutPageContent.espooReservableParagraph": "During the pilot stage you can use the service to reserve the premises, workstations and equipment of the Espoo City Library, Helsinki City Library, the Youth Department, and the Department of Early Education and Care.",
   "AboutPageContent.tampereHeader": "Information about the Varaamo service",
-  "AboutPageContent.tampereText": "<p>Varaamo is an online service through which you can reserve public spaces, workstations and equipment for your use in the Tampere Region.</p><p>The Varaamo system is in implementation phase. The goal of the project is to increase the efficiency of the use of public spaces, improve accessibility and reduce the costs by harmonizing space reservation practices.</p><p>Varaamo is based on the City of Helsinki’s public reservation booking system and since has been developed jointly by the largest municipalities in Finland. In Tampere region, the development of Varaamo is carried out by the City of Tampere's Digitalization Program co-operating with other municipalities in the region.</p><p>The service is still being developed and we are keen to receive any user feedback.</p>",
+  "AboutPageContent.tampereText": "<p>Varaamo is an online service through which you can browse and reserve public spaces, and other resources, such as equipment and market stalls, of the City of Tampere.</p><p>The goal of the service is to increase the efficiency of the use of public spaces, improve accessibility and reduce the costs by harmonizing space reservation practices.</p><p>Varaamo is based on the City of Helsinki’s public reservation booking system. The development of Varaamo is still carried out by the City of Tampere. Your development ideas and feedback are important to us.</p>",
```

Comment:
English text should be slightly different:

```html
<p>
Varaamo is an online service through which you can browse and reserve public spaces as well as other resources such as equipment and market stalls belonging to the City of Tampere.
</p><p>
The goals of the service are to increase efficiency in the use of public spaces, improve accessibility and reduce costs in the harmonization of resource reservation practices. 
</p><p>
Varaamo is based on  the City of Helsinki's public reservation booking system. The City of Tampere is still continuing development of Varaamo.  Your ideas and feedback are important to us.
</p>
```

Reply:
Thanks @danjacob-anders. Updated the English text.